src/SFA.DAS.ApprenticeCommitments.Web/Services/VerifiedUser.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Services
+{
+    public static class VerifiedUser
+    {
+        internal static async Task ConfirmIdentity(HttpContext context)
+        {
+            var identity = (ClaimsIdentity)context.User.Identity;
+            identity.AddClaim(new Claim("VerifiedUser", "True"));
+            await context.SignInAsync(context.User);
+        }
```

Comment:
AddClaim("VerifiedUser"....) is used in a couple of places, would it be worthwhile making this a constant or write an extension method to do this. It's just if someone comes along they might not know the claim is set in two places